browser/src/control/Control.PartsPreview.js

```diff
@@ -393,16 +393,15 @@ L.Control.PartsPreview = L.Control.extend({
 			previewFrameBottom = previewFrameTop + this._previewFrameHeight;
 		}
 
-		var imgSize;
-		if (i === 0 || (previewFrameTop >= topBound && previewFrameTop <= bottomBound)
```

Comment:
Hmm - just a thought - with a 1000 slide file, do we then request all previews ? there is great benefit in requesting only previews around where we are looking at least in the 1st instance - to avoid thrashing a ton of huge images in and scaling them - there can be a large expense here. What is the performance impact ? (didn't read in great detaul but ... ;-)

Reply:
No, it will not request all 1000 preview, it just queue a list. Please read the function
"_processPreviewQueue",  it will request only the visible parts, in your example it will 
request at least 10 previews.